lib/components/views/GotNoVeggies.js

```diff
@@ -0,0 +1,45 @@
+import React, {Component, PropTypes} from 'react';
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+
+import * as authActions from 'actions/authActions';
+import * as userActions from 'actions/userActions';
+
+@connect(
+  state => ({user: state.auth.user}),
+  dispatch => bindActionCreators(authActions, dispatch)
+)
+export default class GotNoVeggies extends Component {
+  static propTypes = {
+    user: PropTypes.object,
+    login: PropTypes.func,
+    store: PropTypes.object
+  };
+
+  static childContextTypes = {
+    muiTheme: React.PropTypes.object
+  }
+
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    let {store} = this.props;
+
+    store.dispatch(userActions.setDonationPreference({donationSize:0, comments:''}));
```

Comment:
The way we are using `bindActionCreators` gives us `setDonationPreference` on props. Getting it off props helps to decouple, and is the pattern we are using elsewhere.